cfgov/unprocessed/js/modules/util/expanded-state.js

```diff
@@ -63,12 +62,26 @@ function toggleExpandedState( $elem, state, cb, delay ) {
   }
 }
 
-module.exports = {
-  get: {
-    isThisExpanded: isThisExpanded,
-    isOneExpanded:  isOneExpanded
-  },
-  set: {
-    toggleExpandedState: toggleExpandedState
+/**
+ * Change the value of the element's aria-expanded attribute based on it's
+ * current state if one is not passed. Allows for a delayed event for
+ * animated elements.
+ * @param {NodeList} elems Element on which to set the aria-expanded attribute.
+ * @param {string} state The value, if any, to set the aria-expanded
+ *   attribute to. Options are "true", "false" or null.
+ * @param {Function} cb The callback function to execute after a delay.
+ * @param {integer} delay The length of time to delay the execution of
+ *                        the passed callback.
+ */
+function toggleGroupExpandedState( elems, state, cb, delay ) {
```

Comment:
Actually, updated in https://github.com/cfpb/cfgov-refresh/commit/02c50006b2657ddf2d639798e7de5943949bd360